app/controllers/staff/reports_controller.rb

```diff
@@ -8,7 +8,20 @@ class Staff::ReportsController < Staff::BaseController
 
   def index
     if current_user.service_owner?
-      reports_for_service_owner
+
+      respond_to do |format|
+        format.html do
+          reports_for_service_owner
+        end
+        format.csv do
+          if downloadable_reports_for_beis_users.map(&:financial_quarter).uniq.length == 1
```

Comment:
I'm wondering whether this should extracted into its own method (`reports_have_same_quarter?`), since it's a little confusing to understand what's happening here..?

Reply:
Good idea! Thanks for the suggestion :-)